app/main/views.py

```diff
@@ -120,10 +131,10 @@ def update(service_id, section):
         )
 
     if form.errors:
+        service.update(form.dirty_data)
```

Comment:
@quis thanks Chris! I've looked at the code a bit more and as far as I understand, what happens right now is:
1. All the price fields are posted as 'priceString', ie 'priceString=<min>&priceString=<max>' ... I still don't quite understand why we need `dirty_data` since